### PR TITLE
ci: Add mergify rules

### PR DESCRIPTION
**Pull request by Madhu-1**
Added Mergify rules to perform below action
* Ask contributor to resolve conflict on the PR if there are any

### CODE REVIEW

**Comment by Madhu-1** [comment]
asking @humblec @nixpanic @ShyamsundarR to review it as I have added your names to auto review requests.

File: .mergify.yml [hunk 1]
```diff
@@ -7,6 +7,22 @@ pull_request_rules:
       dismiss_reviews:
         approved: true
         changes_requested: false
+  - name: ask to resolve conflict
+    conditions:
+      - conflict
+    actions:
+      comment:
+        message: This pull request is now in conflicts. Could you fix it? 🙏
```
Comment (nixpanic) [inline .mergify.yml:15]: Maybe: This pull request now has conflicts with the target branch. Could you please rebase and force push the corrected changes? :pray:

File: .mergify.yml [hunk 2]
```diff
@@ -7,6 +7,22 @@ pull_request_rules:
...
+  - name: Request for review
+    conditions:
+      - base=master
+    actions:
+      request_reviews:
+        users:
```
Comment (nixpanic) [inline .mergify.yml:21]: Not sure about this... should that not be more component based?
We are reviewing PRs quite actively, I think, and adding other reviewers when we expect them to have a look.

Comment (Madhu-1) [reply .mergify.yml]: removed now

**Pull request merged**